src/PixelFlags.cc

```diff
@@ -57,26 +59,70 @@ class FootprintBits : public afw::detection::FootprintFunctor<MaskedImageT> {
 private:
     typename MaskedImageT::Mask::Pixel _bits;
 };
-}  // end anonymous namespace
+typedef std::map<std::string, afw::table::Key<afw::table::Flag>> KeyMap;
+typedef afw::image::MaskedImage<float> MaskedImageT;
```

Comment:
Better not use `MaskedImageT` to refer both to the template parameter above and to the type here. Following coding standard 3-5a, this should be `MaskedImageF`, which has the added benefit of disambiguating them.